bcda/router.go

```diff
@@ -49,7 +48,7 @@ func NewDataRouter() http.Handler {
 	r := chi.NewRouter()
 	m := monitoring.GetMonitor()
 	r.Use(ConnectionClose)
-	r.With(auth.ParseToken, logging.NewStructuredLogger(), auth.RequireTokenAuth,
```

Comment:
We need the token in the context before logging. Without it, `sub`, `aco`, and `token_id` are missing from the request log entries (vs. master, for instance)

Reply:
nice catch!  I can't figure out a good way to test it though...

Reply:
https://github.com/sirupsen/logrus#testing

Reply:
emily created a [ticket](https://jira.cms.gov/browse/BCDA-847)!

Reply:
and i can confirm that locally the token context is being logged:
```
api_1            | {"aco":"0c527d2e-2e8a-4808-b11d-0fa06baf8254","http_method":"GET","http_proto":"HTTP/1.1","http_scheme":"http","level":"info","msg":"request started","remote_addr":"172.21.0.1:38406","sub":"6baf8254-2e8a-4808-b11d-0fa00c527d2e","time":"2019-02-01T20:41:13Z","token_id":"611be673-834a-4069-b4fe-3643a3900508","ts":"Fri, 01 Feb 2019 20:41:13 UTC","uri":"http://localhost:3000/data/100/0c527d2e-2e8a-4808-b11d-0fa06baf8254.ndjson","user_agent":"PostmanRuntime/7.6.0"}
```